src/components/CallView/shared/LocalVideo.vue

```diff
@@ -336,11 +336,12 @@ export default {
 
 .video-container-stripe {
 	position:relative;
-	height: 100%;
 	flex: 0 0 300px;
 	overflow: hidden;
 	display: flex;
 	flex-direction: column;
+	margin-top: 8px;
+	height: 242px !important;
```

Comment:
with 3/4 cameras this skips the rounded corners.
the with is set to 300px (in flex) => /4*3 => 225 => 17 pixels of empty space around it, vs 2*10px of the radius is hardly noticable

![Bildschirmfoto von 2021-11-18 08-48-28](https://user-images.githubusercontent.com/213943/142373790-59aa5cbb-fd48-4cd0-aff5-9d4a59698d68.png)

Could we maybe make the flex grow wider, so the video always fills the height but instead grows by width in the stripe?

Reply:
Should be even worse when someone has a fullHD cam with 16/9 ratio

Reply:
Moved to https://github.com/nextcloud/spreed/issues/6540